File: Back/BancaNet.Backend.Framework/Models/Banks/BankData.cs
using System;

namespace BancaNet.Backend.Framework.Models.Banks
{
	public sealed class BankData : BankBase
	{
		public DateTime? CreatedDate { get; set; }
		public DateTime? UpdatedDate { get; set; }
		public DateTime? BalanceDate { get; set; }
		public decimal? CurrentBalance { get; set; }
		public decimal? LastBalance { get; set; }
		public string CreatorUserID { get; set; }
		public string CreatorUserName { get; set; }
		public string ModifierUserID { get; set; }
		public string ModifierUserName { get; set; }
		public int? DeletionCount { get; set; }
	}
}

File: Front/src/app/models/banks/bank-data.ts
import { BankBase } from './bank-base';

export class BankData extends BankBase {
	public BalanceDate:Date = null;
	public CurrentBalance:number = null;
	public LastBalance:number = null;

	public CreatedDate:Date = null;
	public UpdatedDate:Date = null;
	public CreatorUserID:string = null;
	public CreatorUserName:string = null;
	public ModifierUserID:string = null;
	public ModifierUserName:string = null;

	public DeletionCount:number = 0;

	public FechaCreacion: Date=null;
	public UserName:string =null;
	


}
